repos/rust-lang/ci-mirrors.toml

```diff
@@ -6,3 +6,8 @@ bots = []
 
 [access.teams]
 infra = "write"
+
+[[branch-protections]]
+pattern = "main"
+ci-checks = ["Build finished"]
+required-approvals = 1
```

Comment:
`1` is the default, so no need to spell it out, unless you want to be explicit for some reason :)

Reply:
Didn't know that! Added `dismiss-stale-review = true` in the meantime tho.